src/ImageSharp/Image.Create.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <copyright file="Image.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+
+    using Formats;
+
+    /// <summary>
+    /// Represents an image. Each pixel is a made up four 8-bit components red, green, blue, and alpha
+    /// packed into a single unsigned integer value.
+    /// </summary>
+    public sealed partial class Image
```

Comment:
This feels odd. We're extending the non-generic Image class with Image<TColor> methods. I get that it works, it just feels odd.